src/NuGet.Core/NuGet.LibraryModel/CentralVersionDependency.cs

```diff
@@ -0,0 +1,104 @@
+// Copyright(c) .NET Foundation.All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+using NuGet.Shared;
+using NuGet.Versioning;
+
+namespace NuGet.LibraryModel
+{
+    public class CentralVersionDependency : IEquatable<CentralVersionDependency>, IComparable<CentralVersionDependency>
+    {
+        public string Name { get; }
+
+        public VersionRange VersionRange { get; }
+
+        public CentralVersionDependency(
+            string name,
+            VersionRange versionRange)
+        {
+            Name = name;
+            VersionRange = versionRange;
+        }
+
+        public static implicit operator LibraryRange(CentralVersionDependency library)
```

Comment:
please no :( 
I know other parts of the code use it, but they are unreadable usually. 

@NuGet/nuget-client What do we think about implicit operators?
Personally I'm not a fan cause they mess with readability. 
Whatever we decide here should go in the coding guidelines.

Reply:
Not really needed, removed from here. I would not block the PR on this decision.

Reply:
I'll start a discussion in a PR :)